### ISSUE TITLE
base_conv的Conv2D和fast_conv的Conv2D收敛问题

### ISSUE THREAD

**Issue by d2rivendell** [fmscole/backpropagation#1]
epochs=5
seq=[
    Conv2D(output_channels=1,ksize=5,stride=1),
    Relu(),
    MaxPooling(),
    Conv2D(output_channels=2, ksize=3, stride=1),
    Relu(),
    MaxPooling(),
    Dense(output_num=10),
    Softmax()
]
相同的网络结构， 但是base_conv的Conv2D没有收敛保持在10%左右， fast_conv的Conv2D却收敛了，想请教下这是为什么呢
